examples/examples-gpu/nyc-taxi/rf-scikit.ipynb

```diff
@@ -20,6 +20,13 @@
     "> based on characteristics that can be known at the beginning of a trip, will this trip result in a high tip?"
```

Comment:
<ol><li>Can we point this to the dask-rapids notebook instead of the rapids notebook.  We only have 10 minutes realistically with each trial user, expecting them to get through all 3 notebooks is unlikely.</li><li>Can we duplicate this note above the call to fit which takes like 5 minutes to run?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
After this I'd like to get this merged.  @jameslamb if you have other thoughts on this let's hop in a zoom to save time.

Reply:
sure! I agree with both of those.

Reply:
ok just pushed a commit with these changes. Take another look whenever you have a chance. Once you approve I'll merge this.